src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs

```diff
@@ -220,38 +220,36 @@ private void IO_Completed(object sender, SocketAsyncEventArgs e)
             switch (e.LastOperation)
             {
                 case SocketAsyncOperation.Receive:
-                ProcessReceive(e);
-                break;
+                    ProcessReceive(e);
```

Comment:
Are you using the corefx VS settings? These were moved back when I've imported those in VS.
My personal preference is to have them indented as you have them here (as long as these are spaces and not tabs :) ).